Wikipedia/Code/WMFSearchViewController.m

```diff
@@ -278,9 +278,18 @@ - (void)setSeparatorViewHidden:(BOOL)hidden animated:(BOOL)animated {
 
 #pragma mark - Dismissal
 
-- (IBAction)didTapCloseButton:(id)sender {
+- (BOOL)didRequestDismiss {
     [self.searchField resignFirstResponder];
     [self dismissViewControllerAnimated:YES completion:nil];
+    return YES; // we did dismiss the controller
```

Comment:
Since this is always YES and this is only used for the `accessibilityPerformEscape` method it would be better to make this void and just move it below:

```
- (BOOL)accessibilityPerformEscape {
    [self dismiss];
    return YES;
  }
```